sonar-core/src/main/resources/org/sonar/l10n/core.properties

```diff
@@ -2647,6 +2649,7 @@ branches.last_analysis_date=Last Analysis Date
 branches.no_support.header=Get the most out of SonarQube with branches analysis
 branches.no_support.header.text=Analyze each branch of your project separately with the Developer Edition.
 branches.search_for_branches=Search for branches...
+branches.short_lived.quality_gate.description=The branch status is green because there are no Open issues. The remaining {0} have been Confirmed.
```

Comment:
We never use red/green notation, usually we say "quality gate failed/passed", so I'd suggest to say "The branch quality gate is passed because..."

Also I'd lowercase "open" and "confirmed".